### DISCUSSION TITLE
May I know if any members can merge my new PR?

### DISCUSSION THREAD

**Discussion by gxy5202** [quasarframework/quasar#13221] [Show and Tell]
My PR has been submitted for two days without response...

**Comment by smolinari** [comment]
Be patient. Your PR will be tended to in due time. 

Scott

**Reply by gxy5202** [reply]
> Be patient. Your PR will be tended to in due time.
> 
> Scott

Thanks

**Comment by metalsadman** [comment]
are you referring to this PR https://github.com/quasarframework/quasar/pull/13216 ?
I don't think it's a typo, the sentence is referring to QCarouselSlide's `name` prop. hence why the `s` is not highlighted.

**Reply by gxy5202** [reply]
> are you referring to this PR #13216 ? I don't think it's a typo, the sentence is referring to QCarouselSlide's `name` prop. hence why the `s` is not highlighted.

yes, thank u for your explaination! I misunderstood it. That's fine, it just looks a little weird. I'll close this pr.
